lldb/source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -100,6 +100,7 @@ lldb::REPLSP SwiftREPL::CreateInstanceFromTarget(Status &err, Target &target,
 lldb::REPLSP SwiftREPL::CreateInstanceFromDebugger(Status &err,
                                                    Debugger &debugger,
                                                    const char *repl_options) {
+  printf("Starting the Swift REPL...\n");
```

Comment:
Seems ok since there's a bit of a delay before the REPL is available.

Reply:
Yep, a cold module cache can create some pretty long wait times on slower machines.